python/ml4ir/applications/ranking/tests/test_ranking_model.py

```diff
@@ -93,3 +94,35 @@ def test_csv_and_tfrecord(self):
         # Compare CSV and TFRecord loss and accuracies
         assert np.isclose(tfrecord_loss, csv_loss, rtol=0.01)
         assert np.isclose(tfrecord_mrr, csv_mrr, rtol=0.01)
+
+    def test_linear_ranking_model_save(self):
```

Comment:
Do we also plan to follow-up / have test to load the model @lastmansleeping (in python and jvm side)?

Reply:
Replying on slack due to confidentiality.